File: Gixor-Backend/Gixor-Api/Gixor-Api-GitHub/src/main/java/com/cpy3f2/Gixor/Domain/SimpleUser.java
package com.cpy3f2.Gixor.Domain;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author : simon
 * @description :
 * @last : 2024-11-05 09:23
 * Copyright (c) 2024. 保留所有权利。
 * @since : 2024-11-05 09:23
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class SimpleUser {
    @JsonProperty("id")
    private String githubId;
    @JsonProperty("login")
    private String name;
    @JsonProperty("avatar_url")
    private String avatarUrl;
    @JsonProperty("html_url")
    private String htmlUrl;
}

File: Gixor-Mobile/app/src/main/java/com/cpy3f2/gixor_mobile/model/entity/SimpleUser.kt
package com.cpy3f2.gixor_mobile.model.entity

import com.google.gson.annotations.SerializedName

/**
 * @author : lya
 * @description :粉丝 和 博主
 * @last : 2024-11-05 09:23
 * Copyright (c) 2024. 保留所有权利。
 * @since : 2024-11-05 09:23
 */
data class SimpleUser(
    @SerializedName("id")
    val githubId: String? = null,
    
    @SerializedName("name")
    val name: String? = null,
    
    @SerializedName("avatar_url")
    val avatarUrl: String? = null,
    
    @SerializedName("html_url")
    val htmlUrl: String? = null
)